opm/parser/eclipse/EclipseState/Schedule/UDQFunction.hpp

```diff
@@ -0,0 +1,116 @@
+/*
+  Copyright 2019 Equinor ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef UDQFUNCTION_HPP
+#define UDQFUNCTION_HPP
+
+#include <stdexcept>
+#include <vector>
+#include <functional>
+#include <random>
+
+#include <opm/parser/eclipse/EclipseState/Schedule/UDQSet.hpp>
+
+namespace Opm {
+
+typedef UDQScalar (*scalar_function)(const UDQSet&);
```

Comment:
consider 
``
using scalar_function = UDQScalar (*)(const UDQSet&);
``

Reply:
Thank you - I agree; currently the `typedef` is not in use at all - but I will change to your suggestion, or clear it away completely.